cms/lib/xblock/tagging/tagging.py

```diff
@@ -71,25 +79,36 @@ def save_tags(self, request=None, suffix=None):  # pylint: disable=unused-argume
         """
         Handler to save choosen tags with connected XBlock
         """
-        found = False
-        if 'tag' not in request.params:
-            return Response("The required parameter 'tag' is not passed", status=400)
+        try:
+            posted_data = json.loads(request.body_file.read())
```

Comment:
If you want to take another stab at it, I'll hold off on merging. Otherwise, this is good to go.